generator/test/test_CodeFormatAndQuality.py

```diff
@@ -0,0 +1,39 @@
+"""Interface model unit test
+
+"""
+import subprocess
+import unittest
+from os import walk
+from os.path import join
+from pathlib import Path
+
+from flake8.api import legacy as flake8
+
+
+class CodeFormatAndQuality(unittest.TestCase):
+    def setUp(self):
+        """Searching for all python files to be checked
+
+        """
+        self.list_of_files = []
+        for (directory, _, filenames) in walk(Path(__file__).absolute().parents[1].as_posix()):
+            self.list_of_files += [join(directory, file) for file in filenames
+                                   if file.endswith('.py') and 'test' not in directory
+                                   and 'rpc_spec' not in directory]
+
+    def test_check(self):
```

Comment:
```python3
        """
        Performing checks of Code Format And Quality by flake8
        If any inconvenient low quality code will be found, this will be shown in stdout and
        each such cases will be reflected with report.total_errors number
        """
```